K2Bridge/Factories/AggregateFactory.cs

```diff
@@ -432,6 +436,71 @@ private static PercentileAggregate GetPercentileAggregate(string columnName, str
             return percentileAggregate;
         }
 
+        /// <summary>
+        /// Get tophits aggregate from a given <see cref="DataRow"/>.
+        /// </summary>
+        /// <param name="columnName">The column name.</param>
+        /// <param name="columnMetadata">The column metadata.</param>
+        /// <param name="row">The row to be parsed.</param>
+        /// <param name="logger">ILogger object for logging.</param>
+        /// <param name="query">QueryData containing query information.</param>
+        /// <returns><see cref="TopHitsAggregate"/>.</returns>
+        private static TopHitsAggregate GetTopHitsAggregate(string columnName, string[] columnMetadata, DataRow row, ILogger logger, QueryData query)
+        {
+            logger.LogTrace("Get TopHits aggregate for {}", columnName);
+
+            var random = new Random();
```

Comment:
Nit: creating a random every time is not reccomended, so maybe make this static?

Reply:
Good point. Fixed.